### ISSUE TITLE
The order of RoIAlign and RoIPool params

### ISSUE THREAD

**Issue by Zhu-haow** [open-mmlab/mmcv#809]
I am confused about the order of RoIAlign and RoIPool params

Follow the example you provided in 'test_roi_pool.py' and 'test_roi_align.py'
`rois = (image_id, x_min, y_min, x_max, y_max)`
`froipool = RoIPool((pool_h, pool_w), spatial_scale)`
`froialign = RoIAlign((pool_h, pool_w), spatial_scale, sampling_ratio=2)`
first param is pool_h, which should means vertical axis (i.e height)， and pool_w should means horizontal axis (i.e width).
But , as I test this function as belows:
`x = torch.zeros([1, 1, 15, 15], dtype=torch.float, device='cuda', requires_grad=True)   # feature map`
`x[:,:, 3:5, 6:14] = 1.`
`box = torch.nonzero(x[0][0]).transpose(1, 0)`
`x_min, y_min, x_max, y_max = box[0].min(), box[1].min(), box[0].max(), box[1].max()`
`rois = torch.tensor([[0., y_min, x_min, y_max, x_max]], dtype=torch.float, device='cuda')`
`froipool = RoIPool((pool_h, pool_w), spatial_scale)`
`froialign = RoIAlign((pool_h, pool_w), spatial_scale, sampling_ratio=2)`
`roipool = froipool(x, rois)`
`roialign = froialign(x, rois)`
`print('feature_map:', x)`
`print('box coords: ', rois[0][1:])`
`print('roipooling: ', roipool)`
`print('roialign: ', roialign)
`
I will get the print as below picture.
![image](https://user-images.githubusercontent.com/43101596/106350509-cf17c480-6310-11eb-8f33-0f71b1f70702.png)
It proves that, rois should be (id, y_min, x_min, y_max, x_max) 

I am really confused about x, y, h, w order in this roi_pool and roi_align function and rois. Can you provide a clear example for me. Thanks.

**Comment by ycxioooong** [comment]
In your example codes, you wrote:
```python
box = torch.nonzero(x[0][0]).transpose(1, 0)
x_min, y_min, x_max, y_max = box[0].min(), box[1].min(), box[0].max(), box[1].max()
```

But actually, it should be 
```python
box = torch.nonzero(x[0][0]).transpose(1, 0)
# you first obtain
# [[ y0, x0 ],
#   [ y1, x1 ],
#   ...,
#   [ yN, xN]]
# and then transpose it to
# [[ y0, y1, ..., yN],
#  [ x0, x1, ..., xN]]

# so that
y_min, x_min, y_max, x_max = box[0].min(), box[1].min(), box[0].max(), box[1].max()
```

**Comment by Zhu-haow** [comment]
> In your example codes, you wrote:
> 
> ```python
> box = torch.nonzero(x[0][0]).transpose(1, 0)
> x_min, y_min, x_max, y_max = box[0].min(), box[1].min(), box[0].max(), box[1].max()
> ```
> 
> But actually, it should be
> 
> ```python
> box = torch.nonzero(x[0][0]).transpose(1, 0)
> # you first obtain
> # [[ y0, x0 ],
> #   [ y1, x1 ],
> #   ...,
> #   [ yN, xN]]
> # and then transpose it to
> # [[ y0, y1, ..., yN],
> #  [ x0, x1, ..., xN]]
> 
> # so that
> y_min, x_min, y_max, x_max = box[0].min(), box[1].min(), box[0].max(), box[1].max()
> ```

I got what you mean. Thanks.

**Comment by ycxioooong** [comment]
Cheers!
